.codeclimate.yml

```diff
@@ -8,6 +8,6 @@ plugins:
     enabled: true
   rubocop:
     enabled: true
-    channel: rubocop-0-77
+    channel: rubocop-1-2
```

Comment:
Needs https://github.com/codeclimate/codeclimate-rubocop/issues/262 and https://github.com/puppetlabs/rubocop-i18n/pull/46 to be merged and released

Reply:
Lets use `1.3`